src/interfaces/table.ts

```diff
@@ -0,0 +1,18 @@
+import { ReactNode } from "react";
+
+export interface ColumnType {
+  title: string;
+  key: string;
+  render?: (data: DataType) => ReactNode;
+}
+
+export interface DataType {
+  id: number;
+  name: string;
+  status: string;
+  types: string;
+  quantity: number;
+  price: number;
+  brand: string;
+  [index: string]: number | string;
```

Comment:
why do use `index signature` here?

Reply:
In this case, index signature is not available for values ​​with data type string, so I use index signature which appends string type.